docs/source/webhooks.rst

```diff
@@ -173,22 +173,62 @@ When Not to Use Webhooks
 
 While webhooks are useful, they do have two drawbacks:
 
-* **Not Bidirectional**  - Webhooks simply send data into stackstorm. So if you want data back from   StackStorm, or an action, you'll have to get that data in an asynchronous fashion
-* **No Guarantee of Execution** - Webhooks in StackStorm do not guarantee an execution. This all depends on configuration of the rule that registered the webhook.
+* **Not Bidirectional**  - Webhooks simply submit data into |st2|. So if you want data back from |st2|,
+  or an action execution ID, you'll have to get that data in an asynchronous fashion.
+* **No Guarantee of Execution** - Webhooks in StackStorm do not guarantee an execution. It depends on
+  the configuration of the rule that registered the webhook - based upon the webhook contents, it may
+  not execute any action, or may execute multiple actions.
 
-If you're looking for guaranteed responses, it might be better to directly leverage the
-execution API to directly run actions and query for a response, in the same way the ``st2 run``
-command does behind the scenes. We can get a little insight into this using the ``--debug`` flag:
+If you always want to execute a specific action or workflow, and/or you're looking for guaranteed response,
+you could leverage the ``/v1/executions`` API. This is the same as running an action from the CLI with
```

Comment:
Just a nitpick, really - do we want to say `v1` here? I'm not super plugged in to the upcoming API changes in 2.3 but I assume this will have a new version at some point in the future, so was just planning to refer to it as the "executions API" so we don't have to revisit this. The output below will also contain the correct URL for their reference